frog/models.py

```diff
@@ -501,7 +501,7 @@ def info(self):
             FROG_FFPROBE,
             '-select_streams', 'v:0', '-show_entries', 'stream=width,height,codec_name,duration,avg_frame_rate',
             '-of', 'json',
-            self.source.file.name
+            Path(getRoot()) / self.source.file.name
```

Comment:
getRoot returns a Path object already, no need to cast again

Reply:
Oops!  Thanks for the catch!